src/Kmd.Logic.Cpr.Client/CprClient.cs

```diff
@@ -209,7 +209,7 @@ public async Task<bool> UnsubscribeByIdAsync(Guid id)
         /// <param name="pageNo">The page number to query, starting at 1.</param>
         /// <param name="pageSize">The maximum number of results to return.</param>
         /// <returns>List of citizen records.</returns>
-        public async Task<object> GetAllCprEvents(DateTime dateFom, DateTime dateTo, int pageNo, int pageSize)
+        public async Task<object> GetAllCprEventsAsync(DateTime dateFom, DateTime dateTo, int pageNo, int pageSize)
```

Comment:
This would break any users of the existing version

Reply:
We are changing the version number, until they update version, this will not impact them, this improvement All method are Async, I have made this also Async